CRM/Financial/BAO/FinancialType.php

```diff
@@ -246,6 +246,42 @@ public static function permissionedFinancialTypes(&$permissions, $descriptions)
   }
 
   /**
+   * CRM-21026
+   * Wrapper aroung getAvaliableFinancialTypes to get all including disabled FinancialTypes
+   * @param array $financialTypes
+   *   an array of financial types
+   * @param int|string $action
+   *   the type of action, can be add, view, edit, delete
+   * @param bool $resetCache
+   *   load values from static cache
+   *
+   * @return array
+   */
+  public static function getAllAvailableFinancialTypes($financialTypes = NULL, $action = CRM_Core_Action::VIEW, $resetCache = FALSE) {
```

Comment:
I don't think we should pass in $financialTypes at all - this function should do what it says & get all